README.rst

```diff
@@ -3,3 +3,22 @@ Kebechet
 
 I'm Kebechet, goddess of freshness. I will keep your dependencies fresh and up-to-date.
 
+Deployment of Kebechet
+----------------------
+
+To deploy kebechet on openshift cluster. Follow the procedure:
+
+```
```

Comment:
This is not markdown, but restructured text:
```rst
.. code-block:: console

   echo "Hello!"
```

Reply:
thanks very helpful.